src/lib/utils.js

```diff
@@ -484,11 +484,21 @@ function createDocumentationLink(widget, { connector = false } = {}) {
   ].join('');
 }
 
-function createDocumentationMessageGenerator(widget, { connector } = {}) {
-  const link = createDocumentationLink(widget, { connector });
+function createDocumentationMessageGenerator(
+  widgetOrWidgets,
```

Comment:
what about avoiding this conditional and writing two functions (one plural, one singular, and then do a single pass to put all singular ones into this plural function?)

Reply:
That would also work. The solution in the original post would also remove all conditionals, without writing a new function.

Once all the current PRs are merged (they've all been sent), I can quickly refactor this. WDYT?

Btw I should have originally based all these changes to a specific branch, not `develop`.

Reply:
Good for me. Maybe you could leave this one for last for that reason to avoid too more changes than necessary